docs/user-guide/jobs.md

```diff
@@ -226,6 +228,113 @@ sometimes be started twice.
+| -------------------------------------------------------------------------- |:-----------------:|:---------------------------:|:-------------------:|:-------------------:|
+| [Job Template Expansion](../../examples/job/expansions/README.md)          |                   |                             |          ✓          |          ✓          |
+| [Queue with Pod Per Work Item](../../examples/job/work-queue-1/README.md)  |         ✓         |                             |      sometimes      |          ✓          |
+| [Queue with Variable Pod Count](../../examples/job/work-queue-2/README.md) |                   |         ✓         |             ✓               |                     |          ✓          |
+| Single Job with Static Work Assignment                                     |         ✓         |                             |          ✓          |                     |
+
+This table shows the required settings for `.spec.parallelism` and `.spec.completions`, where `W` is
+the number of work items.
+
+|                             Pattern                                        | `.spec.completions` |  `.spec.parallelism` |
+| -------------------------------------------------------------------------- |:-------------------:|:--------------------:|
+| [Job Template Expansion](../../examples/job/expansions/README.md)          |          1          |     should be 1      |
+| [Queue with Pod Per Work Item](../../examples/job/work-queue-1/README.md)  |          W          |        any           |
+| [Queue with Variable Pod Count](../../examples/job/work-queue-2/README.md) |          1          |        any           |
+| Single Job with Static Work Assignment                                     |          W          |        any           |
+
+<!--
+When you specify completions with `.spec.completions`, each Pod created by the Job controller
+has an identical [`spec`](../devel/api-conventions.md#spec-and-status).  This means that
+all pods will have the same command line and the same
+image, the same volumes, and (almost) the same environment variables.  It is up
+to the user to arrange for the pods to do work on different things.  There are
+several different patterns for parallel jobs, each with strengths and weaknesses.
+The Job object supports   For
+example, the pods might all access a shared work queue service to acquire work
+units.
+-->
+
+### Work Queue with Pod Per Item
+
+The *multiple jobs from template* approach creates one job object for each work item, and so may be
+awkward when there are a very large number of work items.
+
+Also, sometimes you may want to limit the number of pods that are created at the same time, to
+conserve resources, and have the job run until items are completed.
+
+The work-queue-based approaches in this and the following section allow a single Job
+object to manage multiple pods running in parallel, and the number of completions to be
```

Comment:
Unless you're not talking about job's completions, which I assume you don't, I'd rather try to stay away from the word completions. This suggests, you can modify the completions parameter of a job, which we know is not possible.